src/pages/library/datasetBuilder/CohortEditor.ts

```diff
@@ -1,10 +1,8 @@
-import 'rc-slider/assets/index.css';
-
 import _ from 'lodash/fp';
-import Slider from 'rc-slider';
 import React, { Fragment, useState } from 'react';
 import { div, h, h2, h3, strong } from 'react-hyperscript-helpers';
 import { ButtonOutline, ButtonPrimary, GroupedSelect, Link, Select } from 'src/components/common';
+import TerraSlider from 'src/components/common/Slider';
```

Comment:
Nit: suggest naming this "Slider". That's in line with other components like "Select".

Reply:
I agree.